routes/web.php

```diff
@@ -36,7 +38,33 @@
         Route::get('/events',[AdminController::class,'showEvents'])->name('showEvents');
         Route::get('/events/create',[AdminController::class,'createEvent'])->name('createEvent');
         Route::post('/events/store',[AdminController::class,'storeEvent'])->name('storeEvent');
-
+        Route::group(['prefix' => 'chatroom', 'as' => 'chatroom.'], function () {
+            #MEETING
+            Route::group(['prefix' => 'meeting', 'as' => 'meeting.'], function () {
```

Comment:
I suggest using `meetings` instead of `meeting`. It should be in plural form. Please do the same with other routes

Reply:
I changed all group name into plural chatrooms,meetings,rooms,categories